tests/get_kv_table_nodeos_tests.cpp

```diff
@@ -796,6 +955,31 @@ BOOST_FIXTURE_TEST_CASE( get_kv_table_nodeos_test, TESTER ) try {
    chk_result(0, 4);
    chk_result(1, 5);
 
+   // test next_key
+   p.reverse = false;
+   p.index_name = "ff"_n;
+   p.index_value = "";
+   p.encode_type = "dec";
+   p.lower_bound = "0.02";
```

Comment:
Can you add more test cases including negative float numbers?

Reply:
Fixed.